### ISSUE TITLE
Add support for request time in Request Model

### ISSUE THREAD

**Issue by JINDALG** [django-request/django-request#172]
Currently Request model store information only related to Response of URL but it will be good if we can store the time, taken by request also. It will help in analyzing the time taken by different endpoints.

**Comment by JordanHyatt** [comment]
Yes,  this is an obvious feature that should be added,  it will require modifying the middleware to not use the depreciated MiddlewareMixin.  I will submit a PR for this soon

**Comment by JordanHyatt** [comment]
Pull Request #270
